components/containers/tab-bar/tab-bar.tsx

```
@@ -11,6 +11,7 @@ import Animated, {
 import { TabBarButton } from "./tab-bar-button";
 import { HEX_COLORS } from "@/components/ui/gluestack-ui-provider/config";
 import { useSizes } from "@/hooks/useSizes";
+import { Href, router } from "expo-router";
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)